src/Page/Textile.elm

```diff
@@ -85,12 +86,19 @@ type Modal
     = NoModal
     | ComparatorModal
     | AddMaterialModal (Maybe Inputs.MaterialInput) (Autocomplete Material)
+    | ConfirmSwitchToRegulatoryModal
```

Comment:
This seems a bit weird to have such a behaviour on switching tabs. One usually expect tabs to display different informations. But anyway although weird it's clear with the confirmation modal.